pygsheets/cell.py

```diff
@@ -278,7 +296,7 @@ def get_json(self):
             nformat, pattern = self.format, ""
         return {"userEnteredFormat": {
                         "numberFormat": {
-                            "type": nformat.value,
```

Comment:
I have this error before : 'NoneType' object has no attribute 'value' thats why I removed it. Also, self.format always give me a tuple not an enum type.